source/retryable-writes/retryable-writes.rst

```diff
@@ -66,52 +66,9 @@ ClientSession
    name of this object MAY vary across drivers.
 
 Retryable Error
-   An error is considered retryable if it meets any of the following criteria:
-
-   - any network exception (e.g. socket timeout or error)
-
-   - a server error response with any the following codes:
-
-     .. list-table::
-       :header-rows: 1
-
-       * - Error Name
-         - Error Code
-       * - InterruptedAtShutdown
-         - 11600
-       * - InterruptedDueToReplStateChange 
-         - 11602
-       * - NotMaster
-         - 10107
-       * - NotMasterNoSlaveOk
-         - 13435
-       * - NotMasterOrSecondary
-         - 13436
-       * - PrimarySteppedDown
-         - 189
-       * - ShutdownInProgress
-         - 91
-       * - HostNotFound
-         - 7
-       * - HostUnreachable
-         - 6
-       * - NetworkTimeout
-         - 89
-       * - SocketException
-         - 9001
-
-   - a server error response without an error code or one different from those
-     listed above, but with an error message containing the substring "not
-     master" or "node is recovering"
-
-   - a server response with a write concern error meeting the above criteria
-
-   The criteria for retryable errors is similar to the discussion in the SDAM
-   spec's section on `Error Handling`_, but includes additional error codes. See
-   `What do the additional error codes mean?`_ for the reasoning behind these
-   additional errors.
-
-   .. _Error Handling: ../server-discovery-and-monitoring/server-discovery-and-monitoring.rst#error-handling
+   An error is considered retryable if it has a RetryableWriteError label in
+   its top-level "errorLabels" field. See `Determining Retryable Errors`_ for
+   more information.
```

Comment:
@mbroadst Let me know what you think about this and the section below :)

Reply:
LGTM!